nosekatreport/test_template.py

```diff
@@ -1,69 +0,0 @@
-###############################################################################
```

Comment:
@ajoubertza what's your take on the files deleted?
Most of the functions used in this file aren't implemented ie `tests`, `report_subpass`, `report_subfail` and `report_progress`.

Reply:
The report methods look a bit like `Aqf.hop`, `Aqf.fail`, and `Aqf.step`.  You could use those instead.  If these tests weren't being run, then could be removed.

Reply:
Tests weren't being run, also `katscripts` doesn't use any of those methods/functions.